pkgs/development/python-modules/antlr4_9-python3-runtime/default.nix

```diff
@@ -0,0 +1,23 @@
+{ lib, buildPythonPackage, isPy3k, python
+, antlr4_9
+}:
+
+buildPythonPackage rec {
+  pname = "antlr4_9-python3-runtime";
+  inherit (antlr4_9.runtime.cpp) version src;
+  disabled = !isPy3k;
+
+  sourceRoot = "source/runtime/Python3";
+
+  # in 4.9, test was renamed to tests
+  checkPhase = ''
+    cd test*
```

Comment:
Since we have a separate expression for 4.8 and 4.9, just `test` would be better here.